index.html

```diff
@@ -2316,11 +2316,39 @@ <h2>Privacy Considerations</h2>
       <section>
         <h3>Unlinkability</h3>
 
-        <p class="issue">
-When the contents of a digitally signed payload contains correlatable
-identifiers, those identifiers can be used to track individuals. A static
-digital signature is a correlatable identifier. There are digital signature
-schemes that provide uncorrelatable digital signatures.
+        <p>
+When the contents of a digitally signed payload contains a correlatable
+identifier, that identifier can be used for tracking purposes by a verifier,
+which might lead to a violation of an expected level of privacy. An example of
+such an identifier is a shopping loyalty card number. However, some of these
+identifiers might not be immediately apparent, such as a static digital
+signature or a cryptographic hash of an image, which can also be used as a
+correlatable identifier.
+        </p>
+
+        <p>
+It is possible to create a digitally signed payload that does not have any
+correlatable tracking identifiers while also providing some level of assurance
+that the payload is trustworthy for a given interaction. This characteristic is
+called <dfn>unlinkability</dfn> which ensures that no correlatable identifiers
+are used in a digitally signed payload while providing some level of trust.
```

Comment:
```suggestion
are used in a digitally-signed payload while providing some level of trust.
```

Reply:
I feel like we ought to say something like "Verifiers are encouraged to carefully analyze what level of trust they are able to achieve when accepting such payloads." "Some" could also mean "insufficient for the use case".

Reply:
I feel like unlinkability is a potentially dangerous term. it is not solely up to the construction of the credential to provide this feature. there could be ways to construct data in a credential itself that make sharing between colluding verifiers less risky. of course this is up to the issuers to be aware of and using such a scheme.

Reply:
```suggestion
are used in a digitally-signed payload while still providing some level of trust,
the sufficiency of which must be determined by each verifier.
```

Reply:
`unlinkability` is a term used in cryptographic literature to describe a process wherin events that occur, when observed by an attacker, do not appear any more related after the process completes than they did before. Before unlinkability can be claimed, the attacker model needs to be specified, i.e., we would need to be very specific about the context within which we are claiming unlinkability.

Avoidance of correlating identifiers is one aspect of unlinkability, but there is more to it than that, and it may be better to avoid the term, or hedge things by saying stuff like "avoiding correlating identifiers provides some degree of unlinkability"